index.html

```diff
@@ -5,316 +5,515 @@
-                    <p>LET'S CONNECT</p>
+            <div class="partone">
+                <div class="iamtext">
+                    <div class="primarytext">
+                        <h1> I’m Affax, <br> Glad to see you!
+                        </h1>
+                    </div>
+                    <div class="subtext">
+                        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                        </p>
+                    </div>
+                    <div class="socialmedia">
+                        <p>LET'S CONNECT</p>
+                    </div>
+                    <div class="mediaicons">
+                        <ul>
+                            <li><img src="./Images/twitter.svg" alt="T"></li>
+                            <li><img src="./Images/linkedin.svg" alt="L"></li>
+                            <li><img src="./Images/medium.svg" alt="M"></li>
+                            <li><img src="./Images/github.svg" alt="G"></li>
+                            <li><img src="./Images/angellist.svg" alt="A"></li>
+                        </ul>
+                    </div>
                 </div>
-                <ul class="icon">
-                    <li class="first">
-                        <a href="first"><img src="./Images/twitter.svg" alt="twitter"></a>
-                    </li>
-                    <li class="second">
-                        <a href="second"><img src="./Images/Linkedin icon.svg" alt="Linkedin"></a>
-                    </li>
-                    <li class="third">
-                        <a href="third"><img src="./Images/github.svg" alt="github"></a>
-                    </li>
-                    <li class="fourth">
-                        <a href="fourth"><img src="./Images/angellist.svg" alt="angellist"></a>
-                    </li>
-                    <li class="fifth">
-                        <a href="fifth"><img src="./Images/medium.svg" alt="medium"></a>
-                    </li>
-                </ul>
+            </div>
             </div>
         </section>
-        <section class="section2">
-            <div class="container">
-                <div class="photo">
-                    <a href="#"> <img src="./Images/Snapshoot Portfolio.png" alt="SnapshootPortfolio"></a>
-                </div>
-                <div class="leftblock">
-                    <ul class="tags">
-                        <li class="ta1">
-                            <p>html</p>
-                        </li>
-                        <li class="ta2">
-                            <p>css</p>
-                        </li>
-                        <li class="ta3">
-                            <p>javascript</p>
-                        </li>
+        <section class=" listportfolio">
+            <div class="card1">
```

Comment:
- [ ] I think you could use this trick to center the cards and some of the stuff from your hero section. Give it a go. To center things, you can add to them in their css: `margin: 0 auto`. Be careful where you add this though because it will override margins if you have already added some.

yours:
![image](https://user-images.githubusercontent.com/79114233/140363264-5b329bfe-1165-447e-ac13-25f13bf0fbc3.png)

With `margin: 0 auto`:
![image](https://user-images.githubusercontent.com/79114233/140363056-ee5ed21c-26fb-49f1-a1f3-8724b094c148.png)

[OPTIONAL] Try centering other elements from your webpage using this trick, or adding the same padding to both sides of the parent element.

Reply:
hi there. does this apply to the cards or content in the hero section?